backend/runner/runner.go

```diff
@@ -107,6 +117,18 @@ func Start(ctx context.Context, config Config) error {
 	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.RegisterRunner, svc.registrationLoop)
 	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.StreamDeploymentLogs, svc.streamLogsLoop)
 
+	instanceCounter, err = meter.Int64UpDownCounter("ftl.sys.runner.instance",
+		metric.WithDescription("number of runner instances"),
+		metric.WithUnit("{count}"))
+
+	if err != nil {
+		panic(err)
+	}
+
+	moduleNameAttribute := attribute.String("ftl.module.name", "unknown-module")
```

Comment:
This should use your helper function to guarantee consistency, right?

Reply:
yep